src/main/java/core/basesyntax/fruittransact/strategy/FruitTransactionStrategyImpl.java

```diff
@@ -0,0 +1,23 @@
+package core.basesyntax.fruittransact.strategy;
+
+import core.basesyntax.fruittransact.FruitService;
+import core.basesyntax.fruittransact.PurchaseService;
+import core.basesyntax.fruittransact.RemnantService;
+import core.basesyntax.fruittransact.SupplyService;
+
+public class FruitTransactionStrategyImpl implements FruitTransactionStrategy {

```

Comment:
The strategy should look like this:

```
public interface OperationHandler {
    void handle(FruitTransaction transaction);
}

public interface OperationStrategy {
    OperationHandler get(FruitTransaction.Operation operation);
}

public class OperationStrategyImpl implements OperationStrategy {
    private Map<FruitTransaction.Operation, OperationHandler> operationHandlerMap;

    public OperationStrategyImpl(Map<FruitTransaction.Operation,
            OperationHandler> operationHandlerMap) {
        this.operationHandlerMap = operationHandlerMap;
    }

    @Override
    public OperationHandler get(FruitTransaction.Operation operation) {
        return operationHandlerMap.get(operation);
    }
}
```

Reply:
use this suggested implementation instead of a switch case